### ISSUE TITLE
What do I do? When the modal is shown, only the element of this modal can be focus.

### ISSUE THREAD

**Issue by freeflyhm** [yuche/vue-strap#324]
for example, If you press the TAB key, only modal's element can GotFocus.
I can't do it well. Please help me.

**Comment by wffranco** [comment]
What's your question? You only said the modal behavior, not what you want to do.

**Comment by freeflyhm** [comment]
谢谢@wffranco 我的需求是：当打开modal时 只有 modal 里面的元素可以获取焦点，modal 范围外的元素不能获得焦点，就像 bootstrap 那样, [http://getbootstrap.com/javascript/#modals](http://getbootstrap.com/javascript/#modals) 的 demo, 打开模态框后，当我连续按 Tab 键时，焦点始终在模态框内循环，不会跑到模态框外的元素去

**Comment by freeflyhm** [comment]
Only when the element inside the modal open modal can get focus, modal elements cannot be outside the scope of the focus, like bootstrap, http://getbootstrap.com/javascript/#modals demo, open the modal box, when I press the Tab key, the focus has always been in the modal frame cycle, don't run to the outside of the box to modal elements

**Comment by freeflyhm** [comment]
This can be reproduced on the sample page on http://yuche.github.io/vue-strap/#modal

in the Dialog section click 'Show Dialog' button
keep hitting tab key

result: tab going through background page and data can also be entered.

Hope someone can advise on this problem. Thanks.

**Comment by santigarcor** [comment]
Yeah, i reproduced it, in bootstrap the tab key only focus the elements inside the modal, but in vuestrap it focus background elements after focusing modal elements

**Comment by freeflyhm** [comment]
没有解决方案么？ 我暂时在项目中是这样使用的，当 Modal 显示时，用 v-show＝'false'  将 背景后的主页面隐藏起来

期待解决方案。。。
